septentrion/cli.py

```diff
@@ -76,43 +76,21 @@ def split_envvar_value(self, rv):
     is_eager=True,
 )
 @click.option("-v", "--verbose", count=True)
+@click.option("--host", "-H", help="Database host (env: SEPTENTRION_HOST or PGHOST)")
+@click.option("--port", "-p", help="Database port (env: SEPTENTRION_PORT or PGPORT)")
 @click.option(
-    "--host",
-    "-H",
-    help="Database host (env: SEPTENTRION_HOST or PGHOST)",
-    show_default=True,
-    default="localhost",
-)
-@click.option(
-    "--port",
-    "-p",
-    help="Database port (env: SEPTENTRION_PORT or PGPORT)",
-    show_default=True,
-    default=5432,
-)
-@click.option(
-    "--username",
-    "-U",
-    help="Database host (env: SEPTENTRION_USERNAME or PGUSER)",
-    show_default=True,
-    default="postgres",
+    "--username", "-U", help="Database host (env: SEPTENTRION_USERNAME or PGUSER)"
```

Comment:
If you keep a default here then the libpq variables will never be used.
If you want to read PGUSER env var then you are re-doing what is already done by libpq

Reply:
Pass a `default_map` to click containing either:
- the values for lipq envvars when defined, and they will override the static default.
- `None ` for these vars so that there is no default and `connect()` will not pass anything, thus using the default map.

Reply:
http://click.palletsprojects.com/en/7.x/commands/#overriding-defaults

Reply:
I don't understand what is the added value

Reply:
Missed the fact libpq has sane defaults, sorry